identity/src/main/java/com/stripe/android/identity/navigation/IdentityCameraScanFragment.kt

```diff
@@ -78,9 +78,12 @@ internal abstract class IdentityCameraScanFragment(
                         findNavController().navigate(
                             R.id.action_global_couldNotCaptureFragment,
                             bundleOf(
-                                ARG_COULD_NOT_CAPTURE_SCAN_TYPE to identityScanViewModel.targetScanType,
-                                ARG_REQUIRE_LIVE_CAPTURE to verificationPage.documentCapture.requireLiveCapture
-                            )
+                                ARG_COULD_NOT_CAPTURE_SCAN_TYPE to identityScanViewModel.targetScanType
+                            ).also {
+                                if (identityScanViewModel.targetScanType != IdentityScanState.ScanType.SELFIE) {
+                                    ARG_REQUIRE_LIVE_CAPTURE to verificationPage.documentCapture.requireLiveCapture
+                                }
```

Comment:
what does this actually do? IIRC, `also` does not return anything.

Reply:
thanks for spotting this, yes we should do `bundle.putBoolean` instead. fixed and updated tests